CHANGES.rst

```diff
@@ -95,6 +95,9 @@ astropy.tests
 
 - Fixed a bug that meant that the data.astropy.org mirror could not be used when
   using --remote-data=astropy. [#6724]
+- Split pytest plugins into separate modules. [#6384]
```

Comment:
This should probably be `[#6384, #6833]`, right? I'm thinking that'll be useful for someone who sees this and wants to know where to find more context, and this is where the "real" merge happened for this version...

Reply:
this sounds sensible to me, too, though we should add the "manual backport" to #6384, too.